engine/src/test/java/com/redhat/cloud/notifications/processors/camel/CamelTypeProcessorTest.java

```diff
@@ -109,6 +111,7 @@ void afterEach() {
     }
 
     @Test
+    @Transactional
```

Comment:
Is that really necessary?

Reply:
At some point I was actually running SQL in there, but not anymore - removing